.github/workflows/python-package.yml

```diff
@@ -60,7 +60,6 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install pytest
         if [ -f requirements.txt ]; then
-            sed -i 's/>=/==/g' requirements.txt
```

Comment:
This line is the **key part** of the `minimum-build` infrastructure, since it pins the dependencies to the minimum stated version, it should not be removed.